src/components/Footer.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import './CSSmodules/footer.css';
+import { Link } from 'react-router-dom';
+
+class Footer extends PureComponent {
+  render() {
+    return (
+      <>
+        <section className="thesect">
+          <div className="logo">
+            <h1 className="magicians">Math Magicians</h1>
+          </div>
+          <div>
+            <h2 className="maker">Created by Daniel</h2>
+          </div>
+          <nav className="nav">
+            <Link className="links" to="/">Home</Link>
+            |
+            <Link className="links" to="/calculator">Calculator</Link>
+            |
+            <Link className="links" to="/quotes">Quotes</Link>
+          </nav>
```

Comment:
- [ ] Kindly remove links for routes from the footer because it is not good for the user experience and to be consistent also with the wireframe.